app/views/associate_ducr_confirmation.scala.html

```diff
@@ -29,11 +29,14 @@
         headingText = messages("associateDucr.confirmation.heading", flash.get(FlashKeys.DUCR).getOrElse("-"))
     )
 
-    @components.paragraph(messages("movement.confirmation.statusInfo"), Some("status-info"))
+    @components.confirmation_status_info()
 
     <h2 id="what-next">@messages("movement.confirmation.whatNext")</h2>
 
-    @components.paragraph(messages(s"consolidation.confirmation.addOrShut"), Some("next-steps"))
+    <div id="next-steps">
+        <a href="@routes.ChoiceController.startSpecificJourney("associateDUCR")">@messages("consolidation.confirmation.addOrShut.part1")</a> @messages("consolidation.confirmation.addOrShut.part2")
```

Comment:
This enforce structure on translator - i think it would be much better if we would use same solution as title.
Like - we have paragraph main translation and HTML links as parameters.
```
paragraph = Enter {}
link.one = Association
```
and then render would looks like this (this is pseudocode - i'm not sure about API 100%)
```
@messages("paragraph", <a href="routes.Controller.show()">@messages("link.one")</a>)
```
You have full control of text name and it position in paragraph though translation files

Reply:
Changed approach